### DISCUSSION TITLE
[Bug report] ASMB and SMR balances gone after 1.51 upgrade

### DISCUSSION THREAD

**Discussion by Arkiteker** [iotaledger/firefly#2991] [Help & Support]
### Platform

Desktop

### Version

1.51

### Description

I upgraded my Firefly walled to 1.51 and when I signed in my ASMB and SMR balances were WAY less than what I earned from my previous staking.

I had over 7k ASMB and now have only 287+
I had over 2B SMR and now have only 71M+

Even by your own ASMB and SMR calculator my balance of 3.55 Gi should be way more...so WHERE did the rest of my rewards go?

### Expected behaviour

I expected to see the correct ASMB and SMR balances from the previous staking period.

### Actual behaviour

The balances shown are far less than what I had before.

### Can the issue reliably be reproduced?

Yes

### Steps to reproduce the issue

N/A

### Operating System

Windows

### Diagnostics

```shell
views.dashboard.security.version.title
Language: en
Currency: USD
Node selection: Automatic
Platform: win32
Platform Version: 10.0.19043
Platform Architecture: x64
CPU Count: 8
Total Memory: 65209.5 MB
Free Memory: 58130.5 MB
```

### Developer Profile

No

### Profile Type

Software (Stronghold)

### Staking

Actively staking

### Crash Status

None of the above

### Error Messages

N/A

### Error Logs

```shell
Sat, 19 Feb 2022 15:24:15 GMT
StrongholdError: `failed to perform action: `Engine Error: Crypto Error: `error in algorithm XCHACHA20-POLY1305`, Unable to read snapshot. Please try another password.``

Sat, 19 Feb 2022 15:24:07 GMT
StrongholdError: `failed to perform action: `Engine Error: Crypto Error: `error in algorithm XCHACHA20-POLY1305`, Unable to read snapshot. Please try another password.``
```

### Duplicate declaration

- [X] I have searched the issues tracker this issue and there is none

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by nicole-obrien** [comment]
Hey, did you check in your additional wallets? The rewards for staking are per wallet.

https://user-images.githubusercontent.com/89913787/164184564-ec03ae01-bb1e-4df0-8c92-fc0ebd85acb0.mp4

**Comment by Arkiteker** [comment]
How do I move them into a single wallet like I did my mIOTA?

**Comment by nicole-obrien** [comment]
You can not move the rewards until the respective networks are launched. Did you find the rewards in the additional wallets?

**Comment by Alfext** [comment]
I've the same problem. All my Asmb and SMR tokens have dissapeared from FireFly since I updated it to 1.51. In my case, I only see 1 wallet in my FireFly window. I don't know what else I can do. 

views.dashboard.security.version.title
Idioma: es-ES
Divisa: USD
Selección de nodo: Manual
Plataforma: win32
Versión de la plataforma: 10.0.19043
Arquitectura de la plataforma: x64
Número de CPU: 4
Memoria Total: 8078.0 MB
Memoria Libre: 3711.6 MB

**Comment by Alfext** [comment]
I had the same problem, and i've been finally able to solve It.
The key for the solution was this answer I received in GitHub form other user, MikeF05.
"The most likely reason that your SMR and ASMB balance is not displayed is that you can't connect to github. The necessary info is fetched from github. You can check your staking address on [thetangle.org](http://thetangle.org/). You should see your rewards from the first staking period. There will be an alternative way to fetch the info in the next release."
Asking other people smarter than me, we have noticed that in my case, the problem was caused by a filter in the "host" file of my Windows.
Checking that file, we could see that the acces from my computer to the adress "[raw.githubusercontent.com](http://raw.githubusercontent.com/)" was blocked.
As soon as I eliminated that block in my host file, my ASMB and SMR reappeared in my FireFly.
This block could exist not only in the host file of  a Windows O.S., but also in an adblock that anyone coould have installed, or could even be blocked on a DNS level.

Anyone with this issue should try to ping to raw.githubusercontent.com in the first place, and verify if they are or aren´t getting an answer from that address. If not, they should verify what coould be blocking their access to that link, an solve it. The most likely is that solving that block, the ASMB and SMR reapear in their FireFly wallet.

**Reply by armentazz** [reply]
Hello: I have seen this comment as a response to several similar problems, but I have not seen any comments from anyone else that have worked for them.
I have several days looking for the answer and I'm already desperate. I restored my Firefly from seed (I don't have backup addresses) but my Iota balance is 0, I only see first period rewards.
